docs/_static/redirect.js

```diff
@@ -0,0 +1,26 @@
+
+/**
+ * Returns true if the string has a valid URL syntax. 
+ * Does not check whether the page gives 200 or 404 status.
+ */
+function isUrlValid(string) {
+    try {
+        new URL(string);
+        return true;
+    } catch (err) {
+        return false;
+    }
+}
+
+var path = window.location.pathname;
+var page = path.split("/").pop();
+
+BASE_URL = "https://compucell3dreferencemanual.readthedocs.io/en/latest/"; //assume latest
+var newUrl = BASE_URL + "/" + page;
+
+if (isUrlValid(newUrl)) {
+    window.location.href = newUrl;
+}
+else {
+    alert("The PythonScriptingManual is deprecated. Please visit us at " + BASE_URL);
+}
```

Comment:
I tested this locally, but I also acknowledge the possibility of a few 404 errors. I think this is inevitable. The best we can do is check if the URL has valid syntax because trying to check if the web page exists ahead of time violates CORS policy. 
If we choose to rename files inside the ReferenceManual in the future, we would want to include separate redirect scripts for those files. For example:
Renaming `authors.rst` to `writers.rst` would require a short JavaScript embed like this:
```
    window.location.href = "https://compucell3dreferencemanual.readthedocs.io/en/latest/writers.html";
```
...and that JS would be exclusive to that web page. In this hypothetical example, the idea is that a person who bookmarked the PythonScriptingManual version of `authors.html` web page many months ago would then be redirected to the reference manual version of `writers.html`.